addons/editor-theme3/insertion_marker.css

```
@@ -0,0 +1,3 @@
+.blocklyInsertionMarker .blocklyPath.blocklyBlockBackground {
```

This selector applies to all children when it shouldn't.
```suggestion
.blocklyInsertionMarker > .blocklyPath {
```